include/swift/ClangImporter/ClangImporter.h

```diff
@@ -317,6 +317,15 @@ class ClangImporter final : public ClangModuleLoader {
 
 ImportDecl *createImportDecl(ASTContext &Ctx, DeclContext *DC, ClangNode ClangN,
                              ArrayRef<clang::Module *> Exported);
-}
+
+/// Determine whether \c overlayDC is within an overlay module for the
+/// imported context enclosing \c importedDC.
+///
+/// This routine is used for various hacks that are only permitted within
+/// overlays of imported modules, e.g., Objective-C bridging conformances.
+bool isInOverlayModuleForImportedModule(DeclContext *overlayDC,
+                                        DeclContext *importedDC);
```

Comment:
Nitpick: `const`, please.

Reply:
Eh, I guess so. `const` is so not worth it for AST nodes.